.travis.yml

```diff
@@ -6,9 +6,18 @@ jobs:
   include:
     - dist: trusty # Fastest build first
       jdk: oraclejdk8 # Trusty default
-    - jdk: openjdk11 # Bionic default
-    - jdk: openjdk13
-    - os: osx
+    # Whitelist release branches (e.g. v3.5)so they build on all platforms
```

Comment:
we should probably stop using colors for allowlist/denylist

Reply:
Which colors and which allow/deny lists?

Reply:
The comment with `Whitelist` mentioned.
Btw, I've asked CS&S about general concerns on their Slack just so we have some guidance on this from them.  I'm not especially worried because I know human society is slowly changing and becoming "woke" or "aware" of bias in many things.  This is normal, societies improve over time and generally become more caring and as they continue to learn from mistakes from the past.

Anyways, let's not get fully unfocused here.  It's just a comment for now, and let's wait to see what CS&S has to say because it will be helpful guidance for all.

Reply:
Oh right, okay. That's simple enough. @tfmorris could you change that comment to "Allow release branches (e.g. v3.5) to build on all platforms"?

Reply:
It's a USA thing.

@wetneb You have to scroll back up to see it, but the "Outdated" tag shows that this was already changed.